Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -78,6 +78,7 @@ def runSelected(opt):
                      20034.0, #2026D35 ttbar (MTD TDR baseline)
                      20434.0, #2026D41 ttbar (L1T TDR baseline)
                      21234.0, #2026D44 (exercise HF nose)
+                     22034.0, #2026D46 ttbar (exercise V11 HGCal)
```

Comment:
@kpedro88 
can we consolidate some of the exercise this and that workflows some time soon?

Reply:
If we were going to remove one right now, it would probably be D35. At some point, D44 will be replaced with a new version of the HF nose based on D46. But with different geometries that imply code changes, I am reluctant to remove them from regular tests, because things easily break (as we have seen in the past). I agree that we should try not to exceed the current number of workflows (4) if possible.